PWGHF/DataModel/CandidateReconstructionTables.h

```diff
@@ -1272,6 +1344,30 @@ DECLARE_SOA_TABLE(HfCandToOmegaK, "AOD", "HFCANDTOOMEGAK",
                   hf_cand_xic0_omegac0::DecLenCharmBaryon, hf_cand_xic0_omegac0::DecLenCascade, hf_cand_xic0_omegac0::DecLenV0, hf_cand_xic0_omegac0::ErrorDecayLengthCharmBaryon, hf_cand_xic0_omegac0::ErrorDecayLengthXYCharmBaryon,
                   o2::soa::Marker<2>);
 
+// table with results of KFParticle
+DECLARE_SOA_TABLE(HfOmegacKf, "AOD", "HFOMEGACKF", //!
```

Comment:
It is very good to have a separated table for the KF variables. I was wondering though if you need to duplicate some of them, for instance the cosine of pointing angle: is it needed to be there both in the "normal" and KF particle tables? Can't you simply replace the normal cosp with the one computed with the KF, when the KFVertexing is used in place of the DCAFitter?

My suggestion would be to keep the original table as it is, filling it with variables computed with DCAFitter or KFParticle (depending on what is chosen by the user) and having this additional table only for KFParticle specific variables, that do not exist at all with the DCAFitter (for instance all the `Chi2Geo` and `Chi2Topo`). This is what was done for the D0, where only a small additional table is filled in case of KF, see https://github.com/AliceO2Group/O2Physics/blob/b2c8a33a4e8f959849aeb1c2eea40312ba8f4e27/PWGHF/TableProducer/candidateCreator2Prong.cxx#L430-L444

Reply:
Thanks for your suggestion. I delete the duplication from the KF table. The variables I want to fill in the KF table with are those output in the "Omegac0 to Omega Pi" task in Run 2. 
https://github.com/alisw/AliPhysics/blob/8f7fdfc00228ff5f41b02305b26f161534517d48/PWGHF/vertexingHF/AliAnalysisTaskSEXicZero2XiPifromKFP.cxx#L4879-L4931
I think these variables might be useful for machine learning, so I want to save them locally.

Reply:
Thanks! I think you still have these ones that are duplicated:
```
hf_cand_xic0_omegac0::CosPaV0ToPv, hf_cand_xic0_omegac0::CosPaCascToPv,
```

Reply:
The variables you want to store locally should be defined in the tree creator, and can be either in one or the other table produced by the candidate creator.

Reply:
Thanks for your suggestion. It's done.